src/helpers/shareRoom.ts

```diff
@@ -0,0 +1,23 @@
+export default async function shareRoom(): Promise<void> {
+    try {
+        await navigator.clipboard.writeText(location.href);
+
+        let copiedUrlAlert = document.getElementById('copiedUrlAlert')as HTMLSpanElement;
+
+        if(!copiedUrlAlert) {

```

Comment:
commenting this line should break the test, which means your test works when this is implemented